src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/AddTerminalSection.tsx

```diff
@@ -0,0 +1,82 @@
+import { Trans } from '@lingui/macro'
```

Comment:
This component renders this UI:

<img width="488" alt="Screen Shot 2022-07-02 at 7 17 23 PM" src="https://user-images.githubusercontent.com/94939382/176995436-cf492fea-edf9-4bcf-92ed-9efa94e41352.png">